plugins/admission/pkg/admission/admission.go

```diff
@@ -0,0 +1,109 @@
+package admission
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/thoas/go-funk"
+
+	"github.com/fairwindsops/insights-plugins/admission/pkg/models"
+)
+
+var organization string
+var hostname string
+var cluster string
+
+func init() {
+	organization = os.Getenv("FAIRWINDS_ORGANIZATION")
+	hostname = os.Getenv("FAIRWINDS_HOSTNAME")
+	cluster = os.Getenv("FAIRWINDS_CLUSTER")
+}
+
+// SendResults sends the results to Insights
+func SendResults(reports []models.ReportInfo, token string) (bool, []string, []string, error) {
```

Comment:
I'm starting to lean towards naming the return values when there are several or it's not obvious, as a way of self-documenting. Would have saved me from a bug or two in the past, but YMMV

Reply:
Ohh, I actually didn't realize I could do that. I'm totally changing this.